sensum-pwa/src/ui/elements/Navbar.res

```diff
@@ -0,0 +1,19 @@
+@react.component
+let make = (~leftComponent: option<React.element>=?) => {
```

Comment:
How do you read this `~leftComponent: option<React.element>=?`?

Is ok if I say that you receive a prop named `leftComponent` which is React.element type and optional? and the `=?`? Is part of the optional syntax?

Reply:
Why not name it `rightComponent`? I mean you see the gear on the right and if you apply the below suggestion of not using `flex-row-reverse` is a bit more natural, what do you think?

Reply:
it should be `rightComponent` (soy un tortugo!).

> Is ok if I say that you receive a prop named leftComponent which is React.element type and optional? and the =?? Is part of the optional syntax?

- [labeled-arguments](https://rescript-lang.org/docs/manual/latest/bind-to-js-function#labeled-arguments): `~` 
- the type of `leftComponent` is an `option` of `React.element`; So or you have a component or you have `None`.
- `=?` rescript syntax sugar for: [Optional Labeled Arguments](https://rescript-lang.org/docs/manual/latest/function#optional-labeled-arguments)